pycode/test/test_epidata_getCommuterMobility.py

```diff
@@ -329,40 +317,49 @@ def test_some_errors(self, mock_print):
     def test_commuter_data(self):
         """! Tests migration data by some randomly chosen tests.
         """
-        
+
+        df_commuter_migration = gcm.get_commuter_data(out_folder=self.path)
+        # the first column and first row are just the county IDs
+        # mat_commuter_migration is the real Data that should be tested
+        mat_commuter_migration = df_commuter_migration.iloc[: , 0:]
+        mat_commuter_migration = mat_commuter_migration.iloc[0: , :]
+
         countykey2numlist = collections.OrderedDict(
             zip(self.countykey_list, list(range(0, len(self.countykey_list)))))
 
         gD.check_dir(self.path)
         self.write_kreise_deu_data(self.path)
         self.write_commuter_all_federal_states(self.path)
-        self.assertEqual(len(os.listdir(self.path)), 17)
+        self.assertEqual(len(os.listdir(self.path)), 18)
```

Comment:
Why changed this to 18?

Reply:
> Why changed this to 18?

The commuter data is now downloaded in the test-function itself and therefore a 'Germany' folder is created in the fakefilesystem.